packages/vitepress-plugin-git-changelog/src/vite/helpers.ts

```diff
@@ -455,12 +455,29 @@ export function findMapAuthorI18n(mappedAuthor: Contributor): Record<string, str
   return undefined
 }
 
+export function getAvatarFromGithubNoreplyAddress(email: string | undefined, size: number = 80): string | undefined {
+  if (!email)
+    return undefined
+
+  const match = email.match(/^(?:(?<userId>\d+)\+)?(?<userName>[a-zA-Z\d-]{1,39})@users.noreply.github.com$/)
+  if (!match || !match.groups)
+    return undefined
+
+  const { userName, userId } = match.groups
+  return `https://avatars.githubusercontent.com/${userId ? `u/${userId}` : userName}?size=${size}`
```

Comment:
Was it possible to have user's email to match up with their avatar or user profile? I quite forgot how the previous mentioned PRs implemented the feature or discussed. Should we add some context and comments to this function for the newbies to understand what's going on?

And if you could update the corresponding documentations with explainations, it will be awesome!

Reply:
I've added a link to your original comment in this function for context.
What documentation should I update?

Reply:
For docs/pages/ documentations to explain we are now supporting to automatically map to GitHub's no-reply email to obtain the avatar of users.

Reply:
Let quickly merge and finish the documentation in another PR to move faster.